### ISSUE TITLE
Bug number in commits viciously chopped and not hyperlinked

### ISSUE THREAD

**Issue by jidanni** [dear-github/dear-github#179]
I think I found a bug. On
https://github.com/qgis/QGIS/commit/5f6627624ec4711857c5a147c405dac6fb5cfa76
do you see the
 "Fix incorrectly loaded sublayers if they had the same name (fixes #15…

…168)"
at top?
Notice how the bug number is 'viciously' cut in half, and ever more terribly: not hyperlinked to wherever it is supposed to be. (Which in fact turns out to be http://hub.qgis.org/issues/15168 )
Even if it is not chopped (due to not being on a long commit title), please ensure it is still hyperlinked.
OK I'll tell Support that I filed this item...

**Comment by koenpunt** [comment]
I have a feeling this break was already made by the client creating the message

**Comment by Mottie** [comment]
I don't think that issue number will ever get hyperlinked. GitHub only adds hyperlinks to repository issues that exist *on* GitHub, not that external site.
